manifests/0000_30_cluster-api_10_webhooks.yaml

```diff
@@ -49,3 +52,25 @@ webhooks:
         resources:
           - infrastructureproviders
     sideEffects: None
+  - admissionReviewVersions:
+      - v1
+      - v1beta1
```

Comment:
Do we need both versions or should we just use `v1`?

Reply:
In this cluster-api [example](https://github.com/kubernetes-sigs/cluster-api/blob/main/internal/webhooks/cluster.go#L49) they recommend to use both versions.